common/main/nl_BE.xml

```diff
@@ -216,7 +216,6 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 			<nameField type="given" draft="provisional">Juliette</nameField>
 			<nameField type="given-informal" draft="provisional">Juli</nameField>
 			<nameField type="given2" draft="provisional">↑↑↑</nameField>
-			<nameField type="surname" draft="provisional">Van den Berg</nameField>
```

Comment:
@macchiati  is this correct?

Reply:
the err message said surname should not be there

Reply:
Yes, that is correct. I was just creating a PR to do the same thing.

Reply:
It is redundant since we have surname-prefix and surname-core

Reply:
@pedberg-icu thanks.